### PR TITLE
Sources with ang

### PR DESCRIPTION
**Pull request by Gauravjeetsingh**
Adds an object of sources which can be viewed by ang.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -17,6 +17,13 @@ export const SOURCES = {
   R: "Rehatnamas & Panthic Sources"
 };
 
+export const SOURCES_WITH_ANG = {
+  G: SOURCES.G,
+  D: SOURCES.D,
+  B: SOURCES.B,
+  S: SOURCES.S
```
Comment (bogas04) [inline src/index.js]: what about other granths like Nandlal Jee Vaaran, Bhai Gurdas Singh Jee Vaaran, etc?

Comment (Gauravjeetsingh) [reply src/index.js]: This is a list of sources, which we can read by Ang.
And I don't think we can read the remaining Granths by Ang. Can we?

Comment (Gauravjeetsingh) [reply src/index.js]: @bogas04 ping.

**Comment by Gauravjeetsingh** [comment]
@bogas04 ji, I don't have access to merge PR in this repo. If it's all good, can you please go ahead and merge it.

File: src/index.js (hunk 1 above)
Comment (bogas04) [reply src/index.js]: @tarunsingh5 @ManjotS could you please guide us here? Thanks!

Comment (ManjotS) [reply src/index.js]: By design there are a number of sources without page numbers,  which breaks a few use cases, this could be changed, but not here.

**Pull request merged**